openmoc/compatible/opencg_compatible.py

```diff
@@ -387,7 +405,8 @@ def get_compatible_opencg_cells(opencg_cell, opencg_surface, halfspace):
   compatible_cells = list()
 
   # SquarePrism Surfaces
-  if opencg_surface.type == 'z-squareprism':
+  if opencg_surface.type in ['x-squareprism', 'y-squareprism', 
+                             'z-squareprism']:
 
```

Comment:
Could you modify the comments in lines 411-475 to be ambiguous about the plane directions they are referring to?